cmake/pcl_find_libusb.cmake

```diff
@@ -0,0 +1,39 @@
+#
+#pcl_find_libusb is used due to VCPKG making impossible to use local findXX modules.
+# and VCPKG findlibusb doesn't create the libusb targets.
+
+# Find and set libusb
+find_package(libusb)
+
+if(TARGET libusb::libusb)
+  #libusb target is found by the find_package script. 
+  #VCPKG skip PCLs findlibusb and sets its own variables which is handled below.
+  return()
+endif()
+
+#Handle VCPKG definitions
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(libusb DEFAULT_MSG LIBUSB_LIBRARIES LIBUSB_INCLUDE_DIRS)
+
+mark_as_advanced(LIBUSB_INCLUDE_DIRS LIBUSB_LIBRARIES)
+
+if(libusb_FOUND)
+  add_library(libusb::libusb UNKNOWN IMPORTED)
+  set_target_properties(libusb::libusb PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${LIBUSB_INCLUDE_DIRS}")
+  
+  if(LINUX)
+    set_target_properties(libusb::libusb PROPERTIES IMPORTED_LOCATION "${libusb_LIBRARIES}")
+  endif()
+  
+  if(WIN32)
+    foreach(LIBRARY_CONFIG ${LIBUSB_LIBRARIES})
+     if(LIBRARY_CONFIG MATCHES "^(debug|optimized)$")
+       set(_CONF ${LIBRARY_CONFIG})
```

Comment:
Where is `_CONF` used? Due to `elseif` the line below should not use this.

Reply:
Its used in the next "iteration" - first entry is debug or optimized and that is saved in _CONF, then comes the library name, which then depending on _CONF will be added as IMPORTED_LOCATION_(DEBUG).

Reply:
Checked the vcpkg code: I think this loop is unnecessary. You can simply check if the value `LIBUSB_LIBRARY_DEBUG `/`LIBUSB_LIBRARY_RELEASE` is set and set the property values. There is no need to the opposite of [`select_library_configurations`](https://cmake.org/cmake/help/latest/module/SelectLibraryConfigurations.html).